src/mlpack/methods/ann/layer/convolution.hpp

```diff
@@ -73,6 +73,8 @@ template <
 class ConvolutionType : public Layer<MatType>
 {
  public:
+  //! Get the CubeType from MatType
```

Comment:
This is not a very descriptive comment; personally I think it's fine to just leave the line of code without a comment (since the line of code says basically the same thing that the comment does), or, if you want to change to some other comment, that works too.  I'd also add some spacing below to separate this typedef from the empty constructor visually.